src/com/luxoft/java/course/shevtsov/homework/algorithm/additional/NumberSymmetryCheck.java

```diff
@@ -0,0 +1,31 @@
+package com.luxoft.java.course.shevtsov.homework.algorithm.additional;
+
+public class NumberSymmetryCheck {
+    private final static int NUMBER_HALF_LENGTH = 2;
+    public final static int NUMBER_LENGTH = NUMBER_HALF_LENGTH * 2;
+    private static int number1 = 0;
+
+    public static boolean checkNumberSymmetry(int number){
+        String strNumber = "" + number;
```

Comment:
why you convert string to number and then vise verse

Reply:
I convert string to int to pass int number to checkNumberSymmetry() as a parameter (it seems to me it is not right to pass string to the method, which processes number). And in checkNumberSymmetry() I convert int to string to define is a number satisfies the character limit (4 characters). Could you recommend me a better way?

Reply:
4 characters for numbers it's 9999

Reply:
Understood. done